### PR TITLE
Es6 refactor modules in hierarchic directories plugin

### PR DESCRIPTION
**Pull request by rachel-ftw**

### CODE REVIEW

File: lib/ModulesInHierachicDirectoriesPlugin.js [hunk 1]
```diff
@@ -1,46 +1,47 @@
+"use strict";
```
Comment (sokra) [inline lib/ModulesInHierachicDirectoriesPlugin.js]: use script should be after the license comment

File: lib/ModulesInHierachicDirectoriesPlugin.js [hunk 2]
```diff
@@ -1,46 +1,47 @@
...
+		this.target = target;
+	}
+	apply(resolver) {
+		const directories = this.directories;
+		const target = this.target;
+		resolver.plugin(this.source, function(request, callback) {
+			const fs = this.fileSystem;
+			const topLevelCallback = callback;
+			const addrs = getPaths(request.path).paths.map((p) => {
+				return directories.map((d) => this.join(p, d), this);
```
Comment (sokra) [inline lib/ModulesInHierachicDirectoriesPlugin.js]: second argument to `map` doesn't make sense when using arrow function

File: lib/ModulesInHierachicDirectoriesPlugin.js [hunk 3]
```diff
@@ -1,46 +1,47 @@
...
+			}, this).reduce((array, p) => {
```
Comment (sokra) [inline lib/ModulesInHierachicDirectoriesPlugin.js]: second argument to `map` doesn't make sense when using arrow function

**Comment by ooflorent** [comment]
Thank you for your PR but it appears that the file was previously converted to ES6. I'm closing the PR for this reason.

**Pull request closed without merging**